dotCMS/src/main/java/com/dotcms/graphql/business/GraphqlAPIImpl.java

```diff
@@ -256,6 +258,11 @@ private GraphQLSchema generateSchema() throws DotDataException {
         final ContentTypeAPI contentTypeAPI = APILocator.getContentTypeAPI(APILocator.systemUser());
 
         List<ContentType> allTypes = contentTypeAPI.findAll();
+        // exclude ee types when no license
+        if(!isStandardOrEnterprise()) {
+            allTypes = allTypes.stream().filter((type) ->!(type instanceof EnterpriseType))
+                    .collect(Collectors.toList());
```

Comment:
Alternatively could be re-written as:  
`allTypes.stream().filter(EnterpriseType.class::isInstance)`